showcase/app/controllers/components/form/key-value-pair/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const DEFAULT_DATA = [
+  {
+    id: 1,
+    key: 'prod',
+    value: 'This is a production tag',
+  },
+  {
+    id: 2,
+    key: 'enterprise',
+    value: '',
+  },
+  {
+    id: 3,
+    key: 'beta',
+    value: 'Feature includes beta',
+  },
+];
+
+export default class KeyValuePairController extends Controller {
+  @tracked functionalExampleData = DEFAULT_DATA;
+  @tracked canAddRow = this.functionalExampleData.length < 4;
+
+  emptyData = [];
+  sampleDataWith1Row = DEFAULT_DATA.slice(0, 1);
+  sampleData = DEFAULT_DATA;
+
+  @action onKeyInputBlur(item, event) {
```

Comment:
Question: I am not sure what is wrong, but when I blur the input, it throws an error in the console. Does anyone have thoughts on why this would happen?

<img width="1364" alt="Screenshot 2025-06-04 at 4 49 35 PM" src="https://github.com/user-attachments/assets/459e7d70-de8c-4f29-a242-1d72c2800197" />